framework/core/shader_module.cpp

```diff
@@ -42,9 +42,10 @@ inline std::vector<std::string> precompile_shader(const std::string &source)
 		{
 			// Include paths are relative to the base shader directory
 			std::string include_path = line.substr(10);
-			if (!include_path.empty() && include_path.back() == '"')
+			size_t last_delimiter = include_path.find("\"");
```

Comment:
```suggestion
			size_t last_quote = include_path.rfind("\"");
```

Reply:
Hi @carlosadcq , if we use `rfind`, then we would not need to take the substring and then we could remove the magic number?

Reply:
Thanks eloj, I've incorporated the 'last_quote' variable name suggestion. I think for this purposes find or rfind will both work properly since the previous lines of code take care of stripping the first quotes and we would be in trouble if there was more than one quote remaining :)